HHS/Head-Start-TTADP
# [PROD] Fix language bug on existing approved reports

## Description of change

We found a bug when viewing approved reports that don't have language set. This change performs a check before attempting to sort the values.

**NOTE:** This was an urgent fix pushed after hours to prevent issues with viewing approved reports and editing drafts created before the code change.

## How to test

- Edit any draft report created before this code change should allow for save & continue.
- View any v2 approved report, it should load without error.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
